st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -48,6 +52,17 @@ class ActionAliasController(resource.ContentPackResourceController):
         'sort': ['pack', 'name']
     }
 
+    _custom_actions = {
```

Comment:
Actually, it looks like old versions support it as well. Must be something else - looking into it.